### DISCUSSION TITLE
Get error in Make and make install in ubuntu 24.04 LTS AND GMT 6.5

### DISCUSSION THREAD

**Discussion by wajidraki** [gmtsar/gmtsar#1003] [General]
I tried many times but I got this error during installation even though all libraries and depencency installed. I am using Ubuntu 24.04 LTS GMT 6.5 and follow this documentation

http://topex.ucsd.edu/gmtsar/tar/ORBITS.tar
sudo -i
cd /usr/local
mkdir orbits
cd orbits
tar -xvf ~/Downloads/ORBITS.tar # (need full path to ORBITS.tar)      
                                                                                                  
sudo apt-get install csh subversion autoconf libtiff5-dev libhdf5-dev wget
sudo apt-get install liblapack-dev
sudo apt-get install gfortran
sudo apt-get install g++                                        

sudo -i
cd /usr/local
git clone --branch 6.5 https://github.com/gmtsar/gmtsar GMTSAR                
 exit

cd /usr/local
sudo chown -R $USER GMTSAR          

cd GMTSAR
autoconf
autoupdate
./configure --with-orbits-dir=/usr/local/orbits              
   
make
make install
![Screenshot from 2024-11-08 17-23-46](https://github.com/user-attachments/assets/205ddd9b-c1e9-4cf6-94e0-7eb547e55b73)

**Comment by sunset5min** [comment]
input
sudo apt update
sudo apt install build-essential gfortran libfftw3-dev libgdal-dev liblapack-dev

have a try,and hope you will sucess

**Reply by wajidraki** [reply]
Thanks for response problem now resolved
